src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -63,6 +63,9 @@ import {
 
 let nextContentId: number = 0;
 
+/**
+ * Creates individual repeater items.
```

Comment:
I think this should be singular, i.e. `Creates an individual repeater item.` I mean, it's one component per repeater item, right?

Reply:
Correct. Done.